src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -370,3 +376,12 @@ message RpcHttp {
   bool enable_content_negotiation= 16;
   TypedData rawBody = 17;
 }
+
+// Language worker sends dependency download message back to host
+message DependencyDownloadResponse {
+  // unique function identifier
+  string function_id = 1;
+
+  // Result of dependency download operation
+  StatusResult result = 2;  
+}
```

Comment:
Our central repository for the protobuf contract is here: https://github.com/Azure/azure-functions-language-worker-protobuf

Our usual workflow has been making changes to the azure-functions-language-worker-protobuf and then pulling in changes to every other repo (as a git subtree, instructions in README). Could you put a PR there with those changes?

And I think @pragnagopa might have some comments on where/how to add a new proto file in addition to our main FunctionRpc.proto

Reply:
And please let me know if it's easier to sync up in person about this :)

Reply:
Yes there's a separate PR for that https://github.com/Azure/azure-functions-language-worker-protobuf/pull/24, this is pulled as subtree to host, here it's included as for reference

Reply:
Sorry I completely missed that!!